src/com/shashi/service/impl/SellerServiceImpl.java

```diff
@@ -0,0 +1,167 @@
+package com.shashi.service.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.shashi.beans.ProductBean;
+import com.shashi.service.SellerService;
+import com.shashi.utility.DBUtil;
+
+public class SellerServiceImpl implements SellerService {
+
+	@Override
+	public List<ProductBean> getAllProductsBySeller(String companyName) {
+		List<ProductBean> products = new ArrayList<ProductBean>();
+
+		Connection con = DBUtil.provideConnection();
+
+		PreparedStatement ps = null;
+		ResultSet rs = null;
+
+		try {
+			ps = con.prepareStatement("select * from product where sellerId=?");
+			ps.setString(1, companyName);
+			
+			rs = ps.executeQuery();
+
+			while (rs.next()) {
+
+				ProductBean product = new ProductBean();
+
+				product.setProdId(rs.getString(1));
+				product.setProdName(rs.getString(2));
+				product.setProdType(rs.getString(3));
+				product.setProdInfo(rs.getString(4));
+				product.setProdPrice(rs.getDouble(5));
+				product.setProdQuantity(rs.getInt(6));
+				product.setProdImage(rs.getAsciiStream(7));
```

Comment:
why are we missing some columns here? I think we should add them, cause why not. It might cause problems later on